src/Core/IAppBuilderExtensions.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.Owin;
+using Microsoft.Owin.Security;
+using Microsoft.Owin.Security.Cookies;
+using Owin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace RimDev.Stuntman.Core
+{
+    public static class IAppBuilderExtensions
+    {
+        private const string StuntmanAuthenticationType = "StuntmanAuthentication";
+
+        public static void UseStuntman(this IAppBuilder app, StuntmanOptions options)
+        {
+            app.UseCookieAuthentication(new CookieAuthenticationOptions
+            {
+                AuthenticationType = StuntmanAuthenticationType,
+                LoginPath = new PathString(options.SignInUri),
+                LogoutPath = new PathString(options.SignOutUri),
+            });
```

Comment:
If we are going to define `StuntmanOptions.ReturnUrlQueryStringKey` then we should set its value to `CookieAuthenticationOptions.ReturnUrlParameter`. Currently, they are equal, but explicitly setting the latter-value will protect us in case its default changes. Or, refactor to keep cookie options in a variable and just use its value in place of ours.